spec/views/cases/_linked_cases_html_slim_spec.rb

```diff
@@ -51,6 +56,12 @@ def disallow_case_policy(policy_name, kase)
         expect(row.case_type.text).to eq 'FOI '
         expect(row.request.text)
             .to eq "#{ linked_case.subject } #{ linked_case.name }"
+
+        # annoyingly had to add \n below because of how locales using html is handled
```

Comment:
Just to mention, you could also do `row.remove_link.text.strip` to remove whitespace.